resources/route53_hosted_zones.go

```diff
@@ -0,0 +1,389 @@
+						Type:     schema.TypeString,
+						Resolver: schema.PathResolver("GeoLocation.SubdivisionCode"),
+					},
+					{
+						Name: "health_check_id",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "multi_value_answer",
+						Type: schema.TypeBool,
+					},
+					{
+						Name: "region",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "set_identifier",
+						Type: schema.TypeString,
+					},
+					{
+						Name:     "ttl",
+						Type:     schema.TypeBigInt,
+						Resolver: schema.PathResolver("TTL"),
+					},
+					{
+						Name: "traffic_policy_instance_id",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "weight",
+						Type: schema.TypeBigInt,
+					},
+				},
+			},
+			{
+				Name:     "aws_route53_hosted_zone_traffic_policy_instances",
+				Resolver: fetchRoute53HostedZoneTrafficPolicyInstances,
+				Columns: []schema.Column{
+					{
+						Name:     "hosted_zone_id",
+						Type:     schema.TypeUUID,
+						Resolver: schema.ParentIdResolver,
+					},
+					{
+						Name:     "policy_id",
+						Type:     schema.TypeString,
+						Resolver: schema.PathResolver("Id"),
+					},
+					{
+						Name: "message",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "name",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "state",
+						Type: schema.TypeString,
+					},
+					{
+						Name:     "ttl",
+						Type:     schema.TypeBigInt,
+						Resolver: schema.PathResolver("TTL"),
+					},
+					{
+						Name: "traffic_policy_id",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "traffic_policy_type",
+						Type: schema.TypeString,
+					},
+					{
+						Name: "traffic_policy_version",
+						Type: schema.TypeInt,
+					},
+				},
+			},
+			{
+				Name:     "aws_route53_hosted_zone_vpc_association_authorizations",
+				Resolver: fetchRoute53HostedZoneVpcAssociationAuthorizations,
+				Columns: []schema.Column{
+					{
+						Name:     "hosted_zone_id",
+						Type:     schema.TypeUUID,
+						Resolver: schema.ParentIdResolver,
+					},
+					{
+						Name:     "vpc_id",
+						Type:     schema.TypeString,
+						Resolver: schema.PathResolver("VPCId"),
+					},
+					{
+						Name:     "vpc_region",
+						Type:     schema.TypeString,
+						Resolver: schema.PathResolver("VPCRegion"),
+					},
+				},
+			},
+		},
+	}
+}
+
+// ====================================================================================================================
+//                                               Table Resolver Functions
+// ====================================================================================================================
+func fetchRoute53HostedZones(ctx context.Context, meta schema.ClientMeta, parent *schema.Resource, res chan interface{}) error {
+	var config route53.ListHostedZonesInput
+	c := meta.(*client.Client)
+	svc := c.Services().Route53
+
+	processHealthChecksBundle := func(hostedZones []types.HostedZone) error {
```

Comment:
I suggest maybe moving this helper function into route53_types.go, same with health checks